array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,9 @@
+/* exported drop */
+function drop(array, count) {
+  if (array.length === 0) return [];
```

Comment:
Same situation here as above. If the array length is `0`, then your loop will never start, so you end up with the same result